ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/ContractsReader.scala

```diff
@@ -6,47 +6,147 @@ package com.daml.platform.store.appendonlydao.events
 import java.io.InputStream
 import java.time.Instant
 
-import anorm.SqlParser.{binaryStream, int, str}
-import anorm.{Row, RowParser, SimpleSql, SqlParser, SqlStringInterpolation}
+import anorm.SqlParser._
+import anorm.{RowParser, SqlParser, SqlStringInterpolation, _}
 import com.codahale.metrics.Timer
-import com.daml.ledger.participant.state.index.v2.ContractStore
+import com.daml.lf.transaction.GlobalKey
```

Comment:
Isn't that the same as Key within the `events` package?

Reply:
Right, done